app_usb_aud_xk_evk_xu316/src/extensions/audiohw.xc

```diff
@@ -1,233 +1,28 @@
-#include <xs1.h>
-#include <platform.h>
-#include "xassert.h"
 #include "xua.h"
-#include "i2c.h"
-#include "tlv320aic3204.h"
+#include <xk_evk_xu316/board.h>
```

Comment:
These should be quotes rather than angle brackets so it doesn't search the system path

Reply:
The only meaningful difference between "" and <> is that with "" the directory containing the source file will be searched first, with <> it will not.

https://gcc.gnu.org/onlinedocs/gcc-4.1.2/cpp/Search-Path.html  (Its possible clang/xcc search is implemented differently to GCC though)

Reply:
```suggestion
#include "xk_evk_xu316/board.h"
```